### ISSUE TITLE
Autonomic epileptic aura

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6678]
**Preferred term label:**
Autonomic epileptic aura

**Synonyms**
Vegetative aura (remove this from HP:0011154 Focal autonomic seizure, as these are not strict synonyms, the new term allows for Epileptic aura to be inferred from Autonomic epileptic aura, noting that not all Focal autonomic seizures are Epileptic auras - this is an attempt to harmonise concepts from different classifications.)

**Definition (free text, please give PubMed ID)**
A purely subjective clinical manifestation of an epileptic seizure pertaining to the autonomic nervous system.

**Parent term (use hpo.jax.org/app)**
Epileptic aura HP:0033348
Focal autonomic seizure HP:0011154

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
@DLewisSmith  can we provide a comment that explains how this term is different from Focal aware autonomic seizure HP:0032740

https://hpo.jax.org/app/browse/term/HP:0032740

**Comment by DLewisSmith** [comment]
A focal aware autonomic seizure has an initial manifestation pertaining to autonomic nervous system function, which may be objective (for example, pupillary dilation) or subjective (for example, nausea). As a type of focal aware seizure, awareness is retained throughout.

An autonomic epileptic aura is a purely subjective manifestation of an epileptic seizure pertaining to autonomic nervous system function. Consider the following additional more specific annotations: types of 'focal aware autonomic seizure' (https://hpo.jax.org/app/browse/term/HP:0032740) if the aura occurs as the sole manifestation of the seizure, types of 'focal impaired awareness autonomic seizure' (https://hpo.jax.org/app/browse/term/HP:0032755) if the seizure progresses with impairment of awareness, and 'bilateral tonic-clonic seizure with focal onset' (https://hpo.jax.org/app/browse/term/HP:0007334)  if the seizure propagates into a bilateral tonic-clonic seizure.

**Comment by pnrobinson** [comment]
@DLewisSmith  could we add something like this to the definition of Autonomic epileptic aura?

Autonomic auras include cardiorespiratory (e.g., palpitations and shortness of breath), gastrointestinal, genitourinary (genital sensations, urinary urge), and cutaneous (feeling of warmth or cold) sensations. Autonomic symptomatology is produced by activation of the insular cortex, anterior cingulum, SSMA, or amygdala. Abdominal auras constitute the most common type of autonomic aura. These include sensations of nausea, pain, or indescribable discomfort in the abdominal or periumbilical area that can be static, rise to the chest and throat, or descend into the lower abdominal region.  PMID: 20926350

**Comment by DLewisSmith** [comment]
@pnrobinson This looks good. Can we can keep the comment, or at least some explanation of the difference between Autonomic aura and the three concepts of focal autonomic seizure?

**Comment by pnrobinson** [comment]
sure, adding as recommended above and also added the additional explanation.

Where does the synonym "vegatative auras" belong -- probably to this new term?

**Comment by DLewisSmith** [comment]
Yes. I agree

Sent from my phone

On 7 Apr 2021, at 14:13, Peter Robinson ***@***.***> wrote:

﻿

⚠ External sender. Take care when opening links or attachments. Do not provide your login details.

sure, adding as recommended above and also added the additional explanation.

Where does the synonym "vegatative auras" belong -- probably to this new term?

**Comment by pnrobinson** [comment]
thanks! Added this.
